release-process.md

```diff
@@ -319,4 +319,18 @@ Once everything is working (website docs, website changes) create an announcemen
 and then send an e-mail to the mailing list. To create an announcement, create a post under
 `news/_posts` and then run `jekyll build`.
 
+<h4>Update a test suite depending on old releases</h4>
+
+Since Spark always keeps the latest maintance released of each branch in the mirror network,
+we need to update a test suite to check backward compatibility with downloading old versions.
+For now, there is one test suite, `HiveExternalCatalogVersionsSuite`.
+
+For a new maintenance release, `HiveExternalCatalogVersionsSuite` will not fail.
+However, it will cause many trials and fallbacks in all Jenkins builds over corresponding branches.
+This updates should be done as a separate PR after release announcement. (See SPARK-25570)
```

Comment:
I understand what this text is about, but I'm not sure others would get it from this description. Basically, this test suite should refer to only the most recent release from each actively maintained branch, right? I think it's worth giving a short example of what this means.